src/unit_tests/shared/test_syscheck_op.c

```diff
@@ -53,13 +61,20 @@ typedef struct __unescape_syscheck_field_data_s {
     char *output;
 }unescape_syscheck_field_data_t;
 
+typedef struct __registry_group_information {
+    char *name;
+    char *id;
+} registry_group_information;
```

Comment:
We should unify the names of the structures (i.e: `registry_group_information_t` instead of `registry_group_information`)